FEAT #34 EmailService interface and implementation

Kawbat opened the pull request:
Create EmailService interface and EmailServiceImpl, and write test for the implementation. add two dependencies to pom.xml

Kawbat committed:
FEAT #34 EmailService interface and implementation

Create EmailService interface and EmailServiceImpl, and write test for
the implementation. add two dependencies to pom.xml

Xolvez reviewed (changes requested):
Would it be possible to add the SMTP server as an instance variable for the EmailServiceImpl and create it in the constructor?

Kawbat committed:
FEAT  add the SMTP server as an instance variable

add the SMTP server as an instance variable for the EmailServiceImpl and create it in the constructor

Xolvez reviewed (approved):
LGTM
